eng/pipelines/templates/steps/build-test.yml

```diff
@@ -45,7 +46,7 @@ steps:
       foreach ($td in $testDirs) {
         pushd $td
         Write-Host "##[command]Executing go test -run "^Test" -race -v -coverprofile coverage.txt -covermode atomic $td | go-junit-report -set-exit-code > report.xml"
-        go test -run "^Test" -race -v -coverprofile coverage.txt -covermode atomic . | go-junit-report -set-exit-code > report.xml
+        go test -run "^Test" -tags=${{parameters.Tags}} -race -v -coverprofile coverage.txt -covermode atomic . | go-junit-report -set-exit-code > report.xml
```

Comment:
When `Tags` is empty are there any issues with this command? Looks like this is only used for the emulator